src/main/java/com/ecore/roles/client/model/Team.java

```diff
@@ -1,35 +1,21 @@
 package com.ecore.roles.client.model;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
+import lombok.*;
 
-import javax.persistence.Id;
 import java.util.List;
 import java.util.UUID;
 
 @NoArgsConstructor
 @AllArgsConstructor
-@Getter
-@Setter
 @Builder
-@EqualsAndHashCode
+@Data
 public class Team {
 
-    @Id
-    @JsonProperty
```

Comment:
If we aren’t customizing the json serialization, then we don’t need to add the @JsonProperty annotation